packages/kit/src/core/make_package/index.js

```diff
@@ -89,27 +78,27 @@ export async function make_package(config, cwd = process.cwd()) {
 
 		write(path.join(cwd, config.kit.package.dir, out_file), out_contents);
 
-		if (exports_filter(file)) {
-			const entry = `./${out_file}`;
-			package_pkg.exports[entry] = entry;
+		if (!user_defined_exports && exports_filter(file)) {
+			const entry = `./${out_file.replace(/\\/g, '/')}`;
```

Comment:
What's the `replace` thing that had to be added here? Is that for Windows support? I wouldn't expect it's really necessary

Reply:
Spot on, sadly it's necessary because the generated keys for `exports` map would also look like `"./module\\foo\\bar"` and results in a `Missing "./module/foo/bar" export` error.